docs/products/clickhouse/howto/check-data-tiered-storage.rst

```diff
@@ -0,0 +1,81 @@
+Check data distribution between storage devices in Aiven for ClickHouse®'s tiered storage
+=========================================================================================
+
+.. important::
+
+    Aiven for ClickHouse® tiered storage is a :doc:`limited availability feature </docs/platform/concepts/beta_services>`. If you're interested in trying out this feature, contact the sales team at `sales@Aiven.io <mailto:sales@Aiven.io>`_.
+
+Verify how your data is distributed between the two layers of your tiered storage: SSD and object storage.
+
+About checking data distribution
+--------------------------------
+
+If you have the tiered storage feature :doc:`enabled </docs/products/clickhouse/howto/enable-tiered-storage>` on your Aiven for ClickHouse service, your data is distributed between two storage devices (tiers). You can learn on what storage devices specific databases and tables are stored. You can also preview their total sizes as well as part counts, minimum part sizes, median part sizes, and maximum part sizes.
+
+Prerequisites
+-------------
+
+* Aiven organization
+* Access to `Aiven Console <https://console.aiven.io/>`_
+* Tiered storage feature :doc:`enabled </docs/products/clickhouse/howto/enable-tiered-storage>` on an Aiven for ClickHouse service level and on a table level
+* Command line tool (:doc:`ClickHouse client </docs/products/clickhouse/howto/connect-with-clickhouse-cli>`)
+
+Check data distribution in Aiven Console
+----------------------------------------
+
+You can use `Aiven Console <https://console.aiven.io/>`_ to check if the tiered storage is enabled on your service and, if so, how much storage has been used for each type of the disks (the local SSD and the remote object storage) by particular tables.
+
+To check the tiered storage's status information, go to `Aiven Console <https://console.aiven.io/>`_ > your Aiven for ClickHouse service > the **Overview** page > the **Tiered storage** section.
```

Comment:
We're not going to have a TS section in the overview page for Clickhouse, not for the limited availability release, at least, but it may come in for GA.